docs/root/configuration/overview/extension.rst

```diff
@@ -69,6 +69,8 @@ follows:
 Discovery service
 ^^^^^^^^^^^^^^^^^
 
+HTTP Filters
+
 Extension configuration can be supplied dynamically from an :ref:`xDS
```

Comment:
This part should be before the "HTTP Filters" sub-section, as it is generic to all discovery services.